script.js

```diff
@@ -1,28 +1,31 @@
-const msgWaiting = document.querySelector(".message-waiting");
-const searchForm = document.getElementById("search-form");
-const booksDisplay = document.getElementById("books-display");
+const $msgWaiting = $(".message-waiting");
+const $searchForm = $("#search-form");
+const $booksDisplay = $("#books-display");
```

Comment:
Naming the variables starting with a `$` (as in `$msgWaiting`) is not necessary, but a common convention. This convention is used because it quickly lets a developer know that the variables reference jQuery objects, not DOM nodes directly.

This is important, because to interact with these objects we have to use jQuery functions, not vanilla JS DOM functions.

Reply:
Also note that we're using jQuery's CSS-style selector syntax to look up elements in the DOM.